### PR TITLE
GPU Aggregation (8/8): Remove legacy aggregator

### PR DESCRIPTION
**Pull request by Pessimistress**
For https://github.com/visgl/deck.gl/issues/7457

For discussion: there is no more CPUGridLayer/GPUGridLayer, simply `GridLayer`. Should we keep the old layer exports with the `gpuAggregation` flag preset?

This affects the carto module @donmccurdy @felixpalmer  

#### Change List
- Move legacy `AggregationLayer` into the heatmap-layer directory. `HeatmapLayer` is the only remaining layer using it, and we are planning a rewrite.
- Remove unused code and tests

### CODE REVIEW

File: modules/aggregation-layers/src/contour-layer/contour-layer.ts [hunk 1]
```diff
@@ -16,11 +16,11 @@ import {
   DefaultProps
 } from '@deck.gl/core';
 import {PathLayer, SolidPolygonLayer} from '@deck.gl/layers';
-import {WebGLAggregator} from '../aggregation-layer-v9/gpu-aggregator/webgl-aggregator';
-import {CPUAggregator} from '../aggregation-layer-v9/cpu-aggregator/cpu-aggregator';
-import AggregationLayer from '../aggregation-layer-v9/aggregation-layer';
-import {AggregationLayerProps} from '../aggregation-layer-v9/aggregation-layer';
-import {AggregationOperation} from '../aggregation-layer-v9/aggregator';
+import {WebGLAggregator} from '../common/gpu-aggregator/webgl-aggregator';
```
Comment (ibgreen) [inline modules/aggregation-layers/src/contour-layer/contour-layer.ts:19]: Nit: `common` is not an exciting directory name,  Consider splitting into `aggregator` and `aggregation-layer` directories.

One reason why I think this is that the two files are on quite different level of abstractions (i.e. I see the `Aggregator` as a primary export that we would like to continue to make useful for aggregation compute independently of layers.

**Review by ibgreen — APPROVED** [review]

**Comment by donmccurdy** [comment]
If GPUGridLayer has never worked in the v9.x lifecycle then perhaps that isn't needed? But keeping CPUGridLayer as an alias of GridLayer would be a nice backward-compatibility approach I think.

**Comment by ibgreen** [comment]
> For discussion: there is no more CPUGridLayer/GPUGridLayer, simply GridLayer. Should we keep the old layer exports with the gpuAggregation flag preset?

I also think removing it seems reasonable. Probably not the most common layer, and easy enough for affected users to update.

**Comment by felixpalmer** [comment]
I think it is reasonable to remove the old layers, it is straightforward for users to rename the layer

File: modules/aggregation-layers/src/index.ts [hunk 2]
```diff
@@ -19,34 +19,28 @@
...
 export {default as GridLayer} from './grid-layer/grid-layer';
-export {default as GPUGridLayer} from './gpu-grid-layer/gpu-grid-layer';
-export {AGGREGATION_OPERATION} from './utils/aggregation-operation-utils';
-
-// experimental export
 export {default as HeatmapLayer} from './heatmap-layer/heatmap-layer';
-export {default as _GPUGridAggregator} from './utils/gpu-grid-aggregation/gpu-grid-aggregator';
-export {default as _CPUAggregator} from './utils/cpu-aggregator';
-export {default as _AggregationLayer} from './aggregation-layer';
-export {default as _BinSorter} from './utils/bin-sorter';
```
Comment (felixpalmer) [inline modules/aggregation-layers/src/index.ts:34]: @Pessimistress is there some replacement for `BinSorter` in 9.1?

**Pull request merged**